sr_ethercat_hand_config/logging/ws_diff.pl

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/perl
```

Comment:
Another concern that I have is where to place this script. The same concern would apply to bag_rotate.py.

Maybe a package in https://github.com/shadow-robot/common_resources should include these scripts that are going to be used in different types of hands.